docker-compose/generate-config.py

```diff
@@ -104,6 +104,7 @@
       net_back:
     ports:
       - "{buildmaster_wg_ip}:{port}:{port}"
+      - "{buildmaster_pub_ip}:{port}:{port}"
```

Comment:
@vladbogo a cleaner approach would be to make this change only for concerned master (aarch). If you have an idea how to make that, please let me know.